bundle/Core/Helper/ImportUrlsHelper.php

```diff
@@ -0,0 +1,229 @@
+                'totalImported' => $totalImported,
+                'totalUrls' => $totalUrls,
+                'return' => $return,
+                'fileLog' => $this->webDirectory."redirectUrls/logs/redirect_import_urls-".date('d-m-Y').".csv",
+            ];
+
+        }
+
+        return $params;
+    }
+
+
+    public function checkUrlDestinationExist(string $destination):bool
+    {
+        $urlExists = false;
+
+        try {
+            $urlExists = $this->urlWildCardService->translate($destination);
+
+        } catch (\Exception $e) {
+            $this->logger->log(LogLevel::ERROR, $e->getMessage());
+        }
+
+        return $urlExists;
+    }
+
+    public function saveUrls(string $source, string $destination):array
+    {
+        $return = $errors = [];
+        try {
+            $result = $this->urlWildCardService->create($source, $destination, 'Redirection');
+            if ($result) {
+                $return = [
+                    'source' => $source,
+                    'destination' => $destination,
+                    'msg' => $this->translator->trans('nova.import.list.table.info', [], 'redirect'),
+                    'imported' => 'OK',
+                ];
+
+                $msg = $return['msg'];
+                $status = $return['imported'];
+                $this->fs->appendToFile(
+                    $this->webDirectory."redirectUrls/logs/redirect_import_urls-".date('d-m-Y').".csv",
+                    "$source;$destination;$msg;$status\n"
+                );
+            }
+        } catch (\Exception $e) {
+            $return = [
+                'source' => $source,
+                'destination' => $destination,
+                'msg' => $e->getMessage(),
+                'imported' => 'KO',
+            ];
+            $msg = $return['msg'];
+            $status = $return['imported'];
+            $this->fs->appendToFile(
+                $this->webDirectory."redirectUrls/logs/redirect_import_urls-".date('d-m-Y').".csv",
+                "$source;$destination;$msg;$status\n"
+            );
+            $this->logger->log(LogLevel::ERROR, $e->getMessage());
+        }
+
+        return $return;
+    }
+
+    public function saveFileHistory(File $fileToImport, string $fileLog)
+    {
+        try {
+            $redirectImportHistory = new RedirectImportHistory();
+            $redirectImportHistory->setNameFile($fileToImport->getFilename());
+            $redirectImportHistory->setDate(new \DateTime());
+            $redirectImportHistory->setPath($fileLog);
+            $this->entityManager->persist($redirectImportHistory);
+            $this->entityManager->flush();
+        } catch (\Exception $e) {
+            $this->logger->log(LogLevel::ERROR, $e->getMessage());
+        }
+    }
+
+    public function getLogsHistory():array
+    {
+        return $this->entityManager->getRepository("NovaeZSEOBundle:RedirectImportHistory")->findAll();
+    }
+
+    public function log(string $error)
```

Comment:
your function `log` and `trans` do not bring any value to your code. I would remove them and call the service directly.